pkg/controller/tektoninstallation/tektoninstallation_controller_test.go

```diff
@@ -41,14 +45,34 @@ func TestTektonInstallationController(t *testing.T) {
 			require.NoError(t, err)
 
 			AssertThatTektonInstallation(t, tektonInstallation.Namespace, tektonInstallation.Name, cl).
-				HasConditions(InstallationSucceeded())
+				HasConditions(Installing("created tekton subscription"))
 
 			AssertThatSubscription(t, tektonSub.Namespace, tektonSub.Name, cl).
 				Exists().
 				HasSpec(tektonSub.Spec)
 		})
 
 		t.Run("should not requeue", func(t *testing.T) {
+			cl.MockGet = func(ctx context.Context, key client.ObjectKey, obj runtime.Object) error {
+				if _, ok := obj.(*config.Config); ok {
+					installedCode := []config.ConfigCondition{
+						config.ConfigCondition{
+							Code: "applied-addons",
+						},
+						config.ConfigCondition{
+							Code: config.InstalledStatus,
+						},
+						config.ConfigCondition{
+							Code: "validated-pipeline",
+						},
+					}
+					obj = NewTektonConfig(installedCode...)
+					return nil
+				}
+
+				return cl.Client.Get(ctx, key, obj)
+			}
```

Comment:
may I know why do you use the MockGet, instead of setting the `config.Config` as part of the fake client initialization?